Add repeated_primitive_no_list.parquet (#61)
apache/parquet-testing: data/README.md

| column_chunk_key_value_metadata.parquet | two INT32 columns, one with column chunk key-value metadata {"foo": "bar", "thisiskeywithoutvalue": null} note that the second key "thisiskeywithoutvalue", does not have a value, but the value can be mapped to an empty string "" when read depending on the client |
| sorting_columns.parquet | INT64 and BYTE_ARRAY columns with first column with nulls first and descending, second column with nulls last and ascending. This file contains two row groups with same data and sorting columns. |
| old_list_structure.parquet | Single LIST<LIST<INT32>> column with legacy two-level list structure. See [old_list_structure.md](old_list_structure.md) |

> | repeated_primitive_no_list.parquet | INT32 and BYTE_ARRAY REPEATED primitive fields not contained in a LIST annotated group. |

Reviewer: Would you mind adding the following summary of the file, like what we did for `incorrect_map_schema.parquet` and `byte_stream_split.zstd.parquet` above?

```
> parquet-cli meta repeated_no_list.parquet

File path:  repeated_no_list.parquet
Created by: parquet-rs version 53.2.0
Properties: (none)
Schema:
message schema {
  repeated int32 Int32;
  repeated binary String (STRING);
}

Row group 0:  count: 4  65.75 B records  start: 4  total(compressed): 263 B total(uncompressed):263 B
--------------------------------------------------------------------------------
        type      encodings count     avg size   nulls   min / max
Int32   INT32     _ RR_     10        10.90 B    1       "0" / "8"
String  BINARY    _ RR_     10        15.40 B    0       "eight" / "zero"
```

Author: added

> | repeated_primitive_no_list.parquet | INT32 and BYTE_ARRAY REPEATED primitive fields not contained in a LIST annotated group. See [note](#Primitive-REPEATED-fields-with-no-LIST-schema) |

Reviewer: ```suggestion
| repeated_primitive_no_list.parquet | REPEATED INT32 and BYTE_ARRAY fields without LIST annotation. See [note](#REPEATED-primitive-fields-with-no-LIST-annotation) |
```

  }
}
```

> ## Primitive REPEATED fields with no LIST schema

Reviewer: ```suggestion
## REPEATED primitive fields with no LIST annotation
```
